bedrock/firefox/views.py

```diff
@@ -242,6 +242,106 @@ def send_to_device_ajax(request):
     return HttpResponseJSON(resp_data)
 
 
+def firefox_all(request, platform, channel):
```

Comment:
I was thinking it'd be easier to just have separate URL definitions and keep the views separate but because of the l10n component I think you're right that this is better. Can't wait for there to just be one of these URLs.